vulture_os/services/rsyslogd/config/rsyslog_template_sentinel_ama.conf

```diff
@@ -0,0 +1,74 @@
+template(name="sentinel_ama" type="list" option.jsonf="on") {
```

Comment:
you cannot add `option.jsonf="on"` if the output is not json-formatted!
Moreover, this options takes care of json formatting, so you don't need to specify brackets, quotes or whatever
see [documentation](https://www.rsyslog.com/doc/reference/templates/templates-options.html)